src/components/benchmark/VisualizeBenchmarks.tsx

```diff
@@ -1,62 +1,117 @@
 import * as React from 'react';
 import { observer } from 'mobx-react-lite';
-import { Benchmark, BenchmarkValue, BenchmarkTable, sampleBenchmarks } from '../../stores/BenchmarkTypes';
-import { WorkflowSolution } from '../../stores/WorkflowTypes';
+import { BenchmarkValue, BenchmarkRun } from '../../stores/BenchmarkTypes';
 import * as d3 from 'd3';
-import { useStore } from '../../store';
+import './VisualizeBenchmarks.css';
 
 const VisualizeBenchmark: React.FC<any> = observer((props) => {
-  const { exploreDataStore } = useStore();
-  const workflows: WorkflowSolution[] = exploreDataStore.selectedWorkflowSolutions;
-  const benchmarks: Benchmark[] = sampleBenchmarks;
+  const [benchmarkValues, setBenchmarkValues] = React.useState<BenchmarkRun[]>([]);
+  const [expandedRows, setExpandedRows] = React.useState<Record<string, boolean>>({});
 
-  function mapValueToColor(value: number) {
-    const colorScale = d3.scaleSequential()
-      .domain([0, 1])
-      .interpolator(d3.interpolateRdYlGn);
-    const limitRange = d3.scaleLinear()
-      .domain([0, 1])
-      .range([0.2, 0.8]);
-    const color = colorScale(limitRange(value));
-    return color;
+  const mapValueToColor = (value: number) => {
```

Comment:
Yes, I'm using it for the design time benchmarks also. I'd actually like to refactor it into a common module, but it's currently in a different branch (and it would be messy merging those I guess)

Reply:
That's a good idea, we can do it once both are merged into `main`.